src/pools/D3MAavePool.t.sol

```diff
@@ -199,23 +204,36 @@ contract D3MAavePoolTest is D3MPoolBaseTest {
         assertEq(code, 0);
     }
 
+    function testFail_deposit_notHub() public {
+        D3MAavePool(d3mTestPool).deposit(1);
```

Comment:
```suggestion
        D3MTestGem(address(adai)).rely(address(aavePool));
        D3MAavePool(d3mTestPool).deposit(1);
```
Without this addition the test fails anyway.

Reply:
This is a bit weird. It seems the last version doesn't have it but doesn't mark it as outdated. And I haven't changed it lately so not sure why you found it.

Reply:
It was moved to D3MPoolBase.t.sol but I assume the same problem exists there.
It's not so optimal now as the fail tests in the base do not come together with the success tests.

Reply:
Yeah that's a negative trade off. But at least we can enforce every pool will implement those reverts correctly.